### DISCUSSION TITLE
.getFirebaseRoot)().admob is not a function.

### DISCUSSION THREAD

**Discussion by Raynier98** [invertase/react-native-firebase#5373] [Q&A] [answered]
hello. I am making an app. which I am integrating with admods from firebase, but when I tried to add analytics to my project I got the following errors:

this error is when i run pod install on ios:

<img width="572" alt="Captura de Pantalla 2021-05-27 a la(s) 12 17 14 p  m" src="https://user-images.githubusercontent.com/57464993/120002418-cf645b80-bfa2-11eb-81b2-665cbbb9d18a.png">

and error when I run "yarn android" or "ios"

<img width="573" alt="Captura de Pantalla 2021-05-27 a la(s) 12 17 42 p  m" src="https://user-images.githubusercontent.com/57464993/120002423-d0958880-bfa2-11eb-9a56-2974943abaa3.png">

i am new in react native and i need some help

**Comment by mikehardy** [comment, marked as answer]
Admob is no longer a part of firebase upstream and has been removed here in react-native-firebase v12 #5329

**Reply by Raynier98** [reply]
So I have to go back to the previous version to be able to use it?

**Reply by mikehardy** [reply]
https://github.com/invertase/react-native-firebase/issues/5329#issuecomment-843253532
